tasks/deploy/bridge.ts

```diff
@@ -15,23 +18,19 @@ task("Contracts", "Deploys ERC20Safe and the Bridge contract")
     const [adminWallet] = await hre.ethers.getSigners();
     console.log("Admin Public Address:", adminWallet.address);
 
-    const ERC20Safe = await hre.ethers.getContractFactory("ERC20Safe");
-    const safeContract = await ERC20Safe.deploy();
-    await safeContract.deployed();
-    console.log("ERC20Safe deployed to:", safeContract.address);
+    const fs = require("fs");
+    const filename = "setup.config.json";
+    const config = JSON.parse(fs.readFileSync(filename, "utf8"));
 
     const Bridge = await hre.ethers.getContractFactory("Bridge");
-    const bridgeContract = await Bridge.deploy(relayerAddresses, quorum, safeContract.address);
+    let bridgeContract;
```

Comment:
Lines can be merged, perhaps? Also, `const` does not work anymore here?

Reply:
fixed